proto/api/v2/compliance_results_service.proto

```diff
@@ -256,7 +259,7 @@ service ComplianceResultsService {
   }
 
   rpc GetComplianceClusterStats(ComplianceProfileResultsRequest) returns (ListComplianceClusterOverallStatsResponse) {
-    option (google.api.http) = {get: "/v2/compliance/results/profiles/{profile_name}/clusters"};
+    option (google.api.http) = {get: "/v2/compliance/scan/stats/profiles/{profile_name}/clusters"};
```

Comment:
@bradr5 had suggested this elsewhere.  Making the change here as this one is likely to land first.